lib/gltfStatistics.js

```diff
@@ -0,0 +1,202 @@
+                            break;
+                        case 2: // LINE_LOOP
+                            numberOfRenderedPrimitives += indicesCount;
+                            break;
+                        case 3: // LINE_STRIP
+                            numberOfRenderedPrimitives += Math.max(indicesCount - 1, 0);
+                            break;
+                        case 4: // TRIANGLES
+                            numberOfRenderedPrimitives += (indicesCount / 3);
+                            break;
+                        case 5: // TRIANGLE_STRIP
+                        case 6: // TRIANGLE_FAN
+                            numberOfRenderedPrimitives += Math.max(indicesCount - 2, 0);
+                            break;
+                    }
+                }
+                numberOfDrawCalls += primitivesLength;
+            }
+        }
+
+        var children = n.children;
+        var childrenLength = children.length;
+        for (var k = 0; k < childrenLength; ++k) {
+            var child = nodes[children[k]];
+            nodeStack.push(child);
+        }
+    }
+
+    return {
+        numberOfDrawCalls : numberOfDrawCalls,
+        numberOfRenderedPrimitives : numberOfRenderedPrimitives
+    };
+}
+
+function getAllDrawCallStats(gltf) {
+    var numberOfDrawCalls = 0;
+    var numberOfRenderedPrimitives = 0;
+
+    var rootNodes = gltf.scenes[gltf.scene].nodes;
+    var rootNodesLength = rootNodes.length;
+
+    for (var i = 0; i < rootNodesLength; ++i) {
+        var stats = getDrawCallStatistics(gltf, rootNodes[i]);
+        numberOfDrawCalls += stats.numberOfDrawCalls;
+        numberOfRenderedPrimitives += stats.numberOfRenderedPrimitives;
+    }
+
+    return {
+        numberOfDrawCalls : numberOfDrawCalls,
+        numberOfRenderedPrimitives : numberOfRenderedPrimitives
+    };
+}
+
+function gltfStatistics(gltf) {
+    if (!defined(gltf)) {
+        return new Error('gltf must be defined');
+    }
+
+    var drawStats = getAllDrawCallStats(gltf);
+
+    return {
+        buffersSizeInBytes : getBuffersSize(gltf.buffers),
+        numberOfImages : getNumberOfProperties(gltf.images),
+        numberOfExternalRequests : getNumberOfExternalRequests(gltf),
+
+        numberOfDrawCalls : drawStats.numberOfDrawCalls,
+        numberOfRenderedPrimitives : drawStats.numberOfRenderedPrimitives,
+        numberOfNodes : getNumberOfProperties(gltf.nodes),
+        numberOfMeshes : getNumberOfProperties(gltf.meshes),
+        numberOfMaterials : getNumberOfProperties(gltf.materials),
+        numberOfAnimations : getNumberOfProperties(gltf.animations)
+    };
+}
+
+function printStats(gltf, gltfPath) {
+    var stats = gltfStatistics(gltf);
+    var output =
+        '\n' +
+        'Key statistics for ' + gltfPath + '\n' +
+        '--------------\n' +
+        'Total size of all buffers: ' + stats.buffersSizeInBytes.toLocaleString() + ' bytes\n' +
```

Comment:
I'm not sure if there was an original reason for using `toLocaleString` but even just

`Total size of all buffers: ' + stats.buffersSizeInBytes + ' bytes\n'` should be fine